### PR TITLE
color variables for checkbox/radio inset styles

### PR DESCRIPTION
**Pull request by cfarm**
Adds brand colors for the checkbox/radio button Design Manual updates. 

See cf-forms PR for details:

https://github.com/cfpb/cf-forms/pull/41

### CODE REVIEW

File: app/templates/src/static/css/cf-theme-overrides.less [hunk 1]
```diff
@@ -137,6 +137,12 @@
 // .form-label-helper-text
 @label-helper:                  @darkgray;
 
+// .form-l-inset_container
+@input-inset-bg:                @gray-20;
+
+// .form-l-inset_container.is-checked
```
Comment (jimmynotjim) [inline app/templates/src/static/css/cf-theme-overrides.less]: Can this be updated to `:checked` vs a chained class?

**Comment by jimmynotjim** [comment]
:+1: Because my question doesn't actually affect this code, just where this is used.

File: app/templates/src/static/css/cf-theme-overrides.less (hunk 1 above)
Comment (Scotchester) [inline app/templates/src/static/css/cf-theme-overrides.less]: [Not without a polyfill for IE8](http://caniuse.com/#search=%3Achecked)

Comment (jimmynotjim) [inline app/templates/src/static/css/cf-theme-overrides.less]: Why are we styling them at all for IE8? We ought to be enhancing the browsers that support it, and let those that don't use default radios/checkboxes.

Comment (Scotchester) [inline app/templates/src/static/css/cf-theme-overrides.less]: Fair point. If that's easy to do, I support it.

Comment (jimmynotjim) [inline app/templates/src/static/css/cf-theme-overrides.less]: I'd think something like `.lt-ie9 & ` would work just fine.

**Comment by Scotchester** [comment]
:+1:

**Pull request merged**